Another question about the `pkg/volume/flexvolume/flexvolume_test.go` -- when I make the `init` in `execScriptTempl1`, the test fails with

```
=== RUN   TestCanSupport
E0719 13:48:19.829849    2289 driver-call.go:232] init command failed, status: Successx, reason: 
--- FAIL: TestCanSupport (0.00s)
	flexvolume_test.go:180: Can't find the plugin by name
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x11fc5b5]
```

Is that expected that after the first failure, things can just SIGSEGV, or should even the tests be defensive and something like

```
diff --git a/pkg/volume/flexvolume/flexvolume_test.go b/pkg/volume/flexvolume/flexvolume_test.go
index 44af4fc..f47412a 100644
--- a/pkg/volume/flexvolume/flexvolume_test.go
+++ b/pkg/volume/flexvolume/flexvolume_test.go
@@ -177,7 +177,7 @@ func TestCanSupport(t *testing.T) {
        plugMgr.InitPlugins(ProbeVolumePlugins(tmpDir), volumetest.NewFakeVolumeHost("fake", nil, nil))
        plugin, err := plugMgr.FindPluginByName("flexvolume-kubernetes.io/fakeAttacher")
        if err != nil {
-               t.Errorf("Can't find the plugin by name")
+               t.Fatalf("Can't find the plugin by name")
        }
        if plugin.GetPluginName() != "flexvolume-kubernetes.io/fakeAttacher" {
                t.Errorf("Wrong name: %s", plugin.GetPluginName())
```

is called for?

--- a/pkg/volume/flexvolume/unmounter.go
+++ b/pkg/volume/flexvolume/unmounter.go
@@ -51,22 +51,14 @@ func (f *flexVolumeUnmounter) TearDownAt(dir string) error {
 		return nil
 	}
 
-	notmnt, err := isNotMounted(f.mounter, dir)
+	call := f.plugin.NewDriverCall(unmountCmd)
+	call.Append(dir)
+	_, err := call.Run()
+	if isCmdNotSupportedErr(err) {
+		err = (*unmounterDefaults)(f).TearDownAt(dir)
+	}
 	if err != nil {
 		return err
-	}
-	if notmnt {
-		glog.Warningf("Warning: Path: %v already unmounted", dir)
-	} else {
-		call := f.plugin.NewDriverCall(unmountCmd)
-		call.Append(dir)
-		_, err := call.Run()
-		if isCmdNotSupportedErr(err) {
-			err = (*unmounterDefaults)(f).TearDownAt(dir)
-		}
-		if err != nil {
-			return err
-		}
 	}
 
 	// Flexvolume driver may remove the directory. Ignore if it does.

